### PR TITLE
Bug 1938131: pkg/asset/installconfig/aws: Add iam permission for destorying clusters

### PR DESCRIPTION
**Pull request by mtnbikenc**
iam:ListAttachedRolePolicies

Permission required since #4126

### CODE REVIEW

File: pkg/asset/installconfig/aws/permissions.go [hunk 1]
```diff
@@ -118,6 +118,7 @@ var permissions = map[PermissionGroup][]string{
 		"iam:GetRole",
 		"iam:GetRolePolicy",
 		"iam:GetUser",
+		"iam:ListAttachedRolePolicies",
```
Comment (staebler) [inline pkg/asset/installconfig/aws/permissions.go:121]: This should go in `PermissionDeleteBase` instead of `PermissionCreateBase`.

Comment (mtnbikenc) [reply pkg/asset/installconfig/aws/permissions.go]: Updated.

**Review by staebler — APPROVED** [review]
/lgtm

**Pull request merged**